src/ecosystem/corals/Corals.jl

```diff
@@ -35,19 +35,36 @@ function colony_mean_area(colony_diam_means::Array{T})::Array{T} where {T<:Float
     return pi .* ((colony_diam_means ./ 2.0) .^ 2)
 end
 
+"""
+    linear_extensions()
+
+Linear extensions. The values are converted from `cm` to the desired unit.
+The default unit is `m`.
+"""
+function linear_extensions(; unit=:m)::Matrix{Float64}
```

Comment:
Waiting on further clarification as to why the unit conversion is necessary, and why meters over centimeters

Reply:
When we convert from relative to absolute cover, we use `habitable_loc_areas′`. This area comes from `domain.site_data.area`. We can use `linear_extension` and `bin_edges` in cm, but then we also need to convert this area to cm^2.

Reply:
Lets resolve in a separate PR then